Sources/SwiftKuery/QueryBuilder.swift

```diff
@@ -87,16 +120,24 @@ public class QueryBuilder {
     /// DateFormatter to convert between Date and String instances.
     public let dateFormatter: DateFormatter?
 
-    /// Initialize an instance of QueryBuilder.
-    ///
-    /// - Parameter addNumbersToParameters: An indication whether query parameters should be numbered.
-    /// - Parameter firstParameterIndex: The starting index for numbered parameters.
-    /// - Parameter anyOnSubquerySupported: An indication whether ANY on subqueries is supported.
-    /// - Parameter withDeleteRequiresUsing: An indication whether a `DELETE` query should use `USING` clause for tables in `WITH` clause.
-    /// - Parameter withUpdateRequiresFrom: An indication whether an `UPDATE` query should use `FROM` clause for tables in `WITH` clause.
-    /// - Parameter createAutoIncrement: A function to create column's autoincrement expression based on the column's type.
-    /// - Parameter dropIndexRequiresOnTableName: An indication whether the drop index syntax requires `ON table.name` clause.
-    /// - Parameter dateFormatter: DateFormatter to convert between Date and String instances.
+    // MARK: Initializer
+    /**
+     Initialize an instance of QueryBuilder.
+     ### Usage Example: ###
+     In this example, a `QueryBuilder` for PostgreSQL is initialized. Parameters not defined are set to default values.
+     ```swift
+     let queryBuilder = QueryBuilder(withDeleteRequiresUsing: true, withUpdateRequiresFrom: true, createAutoIncrement: createAutoIncrement)
+     ```
+    
+     - Parameter addNumbersToParameters: An indication whether query parameters should be numbered.
+     - Parameter firstParameterIndex: The starting index for numbered parameters.
+     - Parameter anyOnSubquerySupported: An indication whether ANY on subqueries is supported.
+     - Parameter withDeleteRequiresUsing: An indication whether a `DELETE` query should use `USING` clause for tables in `WITH` clause.
```

Comment:
Please read through the parameters list as some need "the" adding in to make the definitions read as sentences e.g. "should use the `USING` clause...."